sprint7-js-intro/ex1.js

```diff
@@ -0,0 +1,24 @@
+/*Veți crea o funcție care vă va returna un boolean „adevărat” sau „fals” ca rezultat. Numărul introdus ar trebui să returneze un „adevărat” doar dacă este divizibil cu 10.
+În caz contrar, programul dvs. ar trebui să returneze un răspuns „fals”.
+
+Exemplu 1:
+Input: 100
+
+Output: true
+
+Exemplu 2:
+
+Input: 23
+
+Output: false*/
+
+function DivizibilCu10(numar) {
```

Comment:
```suggestion
function divizibilCu10(numar) {
```

ca functii acceptam formate doar snake_case sau camelCase